src/backend/assets/js/main.js

```diff
@@ -1,5 +1,87 @@
 'use strict';
 
+function handleClientLoad() {
+  gapi.load('client:auth2', initClient);
+}
+
+function initClient() {
+  let id = document.getElementById('gcalendar-id').value;
+  let key = document.getElementById('gcalendar-key').value;
+  let CLIENT_ID = id;
+  let API_KEY = key;
+  let DISCOVERY_DOCS = ["https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest"];
+  let SCOPES = "https://www.googleapis.com/auth/calendar";
+
+  gapi.client.init({
+    apiKey: API_KEY,
+    clientId: CLIENT_ID,
+    discoveryDocs: DISCOVERY_DOCS,
+    scope: SCOPES
+  }).then(function() {
+    gapi.auth2.getAuthInstance().isSignedIn.listen(updateSigninStatus);
+    updateSigninStatus(gapi.auth2.getAuthInstance().isSignedIn.get());
+  });
+}
+
+function updateSigninStatus(isSignedIn) {
+  if (isSignedIn) {
+    console.log("Signed IN");
```

Comment:
@agbilotia1998 I don't know if this console.log is written for testing or is it needed in the code.

Reply:
Oh, was just checking if session storage is effected by changing client ID. Thanks for pointing out :sweat_smile: